pkg/cmd/server/api/types.go

```diff
@@ -144,6 +144,24 @@ type ProjectConfig struct {
 	// It is in the format namespace/template and it is optional.
 	// If it is not specified, a default template is used.
 	ProjectRequestTemplate string
+
+	// SecurityAllocator controls the automatic allocation of UIDs and MCS labels to a project. If nil, allocation is disabled.
```

Comment:
If the allocator is off it just means your pods will fail to start with an error "no uid".  We'll want to deal with that in the SCC changes.

----- Original Message -----
> > @@ -144,6 +144,24 @@ type ProjectConfig struct {
> >  	// It is in the format namespace/template and it is optional.
> >  	// If it is not specified, a default template is used.
> >  	ProjectRequestTemplate string
> > +
> > +	// SecurityAllocator controls the automatic allocation of UIDs and MCS
> > labels to a project. If nil, allocation is disabled.
> 
> > Just means the allocator is disabled.  We still need to define the security
> > mode config, but it would be manual allocation vs automatic (perhaps by
> > making the template have a common range and then moving projects out to
> > their own blocks as needed)
> 
> Will we have a story for this before we ship?  If not, we probably shouldn't
> allow it to be missing.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2893/files#r31913792